testing/MLDB-1209_statstable_bias_noise_test.py

```diff
@@ -0,0 +1,132 @@
+#
+# MLDB-1209_statstable_bias_noise_test.py
+# Francois Maillet, 2016-07-04
+# This file is part of MLDB. Copyright 2016 Datacratic. All rights reserved.
+#
+
+import datetime
+import random
+
+mldb = mldb_wrapper.wrap(mldb)  # noqa
+
+class MLDB1209StatstableBiasNoiseTest(MldbUnitTest):  # noqa
```

Comment:
I had not realized that this is also lacking some validation point.  I will add these too.